home/README.md

```diff
@@ -0,0 +1,10 @@
+# isa7-css-ecommerce-project
+## CSS Assignment - 10 points
+- Instructions (you can design an ecommerce website the you like but make sure it has the following)
+   - products (2 points) 
+   - product details (2 points)
+   - cart + payment details (2 points)
+   - registration, login functionality (2 points)
+   - deploy the project on netlify and share the deployment link in README.md; do not forget to make a PR as well (1 point)
+   - Your creativity and additional functionality to the project will help you to get extra 1 points
+  
```

Comment:
Check the structure of the files again, keep home page on the first level, create a single folder for the assets, check the page's links, and try to deploy again if needed.
Work on responsiveness today and we can review them tomorrow during our 1o1.